micromanager_gui/multid_widget.py

```diff
@@ -273,6 +275,17 @@ def add_position(self):
                 item.setTextAlignment(int(Qt.AlignHCenter | Qt.AlignVCenter))
                 self.stage_tableWidget.setItem(idx, c, item)
 
+            if self._mmc.getAutoFocusDevice() and self._mmc.isContinuousFocusEnabled():
+                autofocus_dev = AutofocusDevice.set(
+                    self._mmc.getAutoFocusDevice(), self._mmc
+                )
+                autofocus_dev.get_position()
```

Comment:
is this doing anything?

Reply:
@tlambert03  This is only adding the `z offset` to the position table if the z offset is engaged. Nothing in particular at the moment since there is no offset info in `useq-schema`/`pymmcore-plus` `run_mda`.

Reply:
but `autofocus_dev.get_position()` gets the position and doesn't assign it to anything?

Reply:
Oh I see, you are right, line 282 is not used!